content/docs/setup/kubernetes/install/operator/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Operator Install 
+description: Install and configure Istio using the Istio Operator.
+weight: 25
+keywords: [operator,kubernetes,helm]
+---
+
+Follow this guide to install and configure an Istio mesh using an alternate
+installation method: the Istio {{<gloss operator>}}Operator{{</gloss>}}
+installation.
+
+The Istio Operator installation process has several benefits:
+
+- Common install tasks require fewer steps, some are a simple one-line command.
+
+- Strong user input validation to help prevent installation errors.
+
+- Rich customization options to easily override any aspect of the configuration.
+
+- User-friendly error messages to spot configuration errors easier.
+
+- Smart installer capable of waiting for Istio pods and services to be in a
+  ready state.
+
+- Fewer dependencies (doesn’t require Helm).
+
+The Operator install is accessed via [istioctl](/docs/ops/setup/istioctl/)
+commands and a new Operator API.
+
+{{< tip >}}
+The Istio Operator is a replacement for Helm templates and Tiller
+installation/operation.
+{{< /tip >}}
```

Comment:
not a replacement (yet).  I'd suggest just removing this tip.

Reply:
addressed